adagrams/game.py

```diff
@@ -1,11 +1,103 @@
+import copy
+import random
+    # create a list (ORIGINAL_POOL) with all the given letters
+    #LOOP:
+        # create deep copy of ORIGINAL_POOL (GAME_POOL)
+        # use random.choice(list) function to draw from pool (GAME_POOL) of letters
+        # make another list for player's hand (HAND)
+        # once letter is picked remove letter from pool list
+        # use while loop max the letters of player's hand to 10
+
 def draw_letters():
-    pass
+    ORIGINAL_POOL = ["A", "A", "A", "A", "A", "A", "A", "A", "A", "B", "B", "C", "C",
+    "D", "D", "D", "D", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E",
+    "F", "F", "G", "G", "G", "H", "H", "I", "I", "I", "I", "I", "I", "I", "I", "I", 
+    "J", "K", "L", "L", "L", "L", "M", "M", "N", "N", "N", "N", "N", "N",
+    "O", "O", "O", "O", "O", "O", "O", "O", "P", "P", "Q", "R", "R", "R", "R", "R", "R", 
+    "S", "S", "S", "S", "T", "T", "T", "T", "T", "T", "U", "U", "U", "U", 
+    "V", "V", "W", "W", "X", "Y", "Y", "Z"]
```

Comment:
I like that this implementation accurately represents the probability of choosing a certain letter from the pool of letters. Is there a way that we could have made this list of strings programmatically? 🤔 Something to consider!